pallets/snarcos/src/benchmarking/suite.rs

```diff
@@ -103,4 +106,7 @@ benchmarks! {
         let _ = insert_key::<T>(key);
     } : verify(caller::<T>(), IDENTIFIER, proof, input, Marlin)
 
+    // Benchmarks as unit tests
+
```

Comment:
nit: unnecessary blank link

Reply:
yeah, but it is consistent with other such 'section-markers' within this long suite so I think I will left it here